contracts/TablelandTables.sol

```diff
@@ -146,24 +145,14 @@ contract TablelandTables is
         uint256 tableId,
         address controller
     ) external override whenNotPaused {
-        // table doesn't exist or
```

Comment:
I don't normally like hard to read code like this, but i think it makes sense in solidity where contract size is a concern. The interface doc string has a good overview of all these checks, which makes me feel okay about it.